packages/bpk-component-nudger/src/BpkNudger.js

```diff
@@ -106,10 +114,12 @@ BpkNudger.propTypes = {
   value: PropTypes.number.isRequired,
   onChange: PropTypes.func.isRequired,
   className: PropTypes.string,
+  buttonType: PropTypes.string,
```

Comment:
```suggestion
  buttonType: PropTypes.oneOf(['secondary', 'default']),
```

Reply:
I would probably change it to `oneOf(['secondary', 'outline'])`, wdyt?

Reply:
Yeah sorry, I mean to write `outline` but wrote `default` for some reason 🤦‍♂

Reply:
should be updated 😄